### PR TITLE
Us119 display charging clause

### PR DESCRIPTION
**Pull request by Kirosoft**
- do not merge \* - waiting acceptance tests

### CODE REVIEW

File: application/templates/viewdeed.html [hunk 1]
```diff
@@ -1,84 +1,86 @@
...
+        <dl class="definition-inline">
+          <dt class="bold-text">Title number</dt>
+          <dd>{{ deed_data.deed.title_number }}</dd>
+        </dl>
 
-          <h3 class="heading-medium">Lender</h3>
+        <h3 class="heading-medium">Lender</h3>
         <div class="text">
-            Lender Information will go here
+          <p>{{ deed_data.deed.lender.name }}<br/>
```
Comment (endamccormack) [inline application/templates/viewdeed.html:28]: all 3 of these should probably be in their own paragraphs, semantically they are different things

**Comment by endamccormack** [comment]
:+1:

**Comment by danielmarker** [comment]
:+1:

File: application/templates/viewdeed.html [hunk 2]
```diff
@@ -1,84 +1,86 @@
...
-          <p class="bordered-output-not-present">[Effective date and time will be shown here]</p>
-          <label class="heading-small" for="signature">Borrower's electronic signature(s)</label>
+        <p>This charge takes effect when the registrar receives notification from
+          Pure Law that the charge is to take effect.</p>
+        <p class="bordered-output-not-present">[Effective date and time will be shown here]</p>
+        <label class="heading-small" for="signature">Borrower's electronic signature(s)</label>
         {% for borrower in deed_data.deed.borrowers %}
-          <p class="bordered-output-not-present" id="signature{{loop.index}}">[e-signature of {{ borrower.forename}} {{ borrower.middle_name}}{% if borrower.middle_name %} {% endif %}{{
-            borrower.surname}} goes here]</p>
+          <p class="bordered-output-not-present" id="signature{{ loop.index }}">[e-signature
```
Comment (danielmarker) [inline application/templates/viewdeed.html:62]: Like this :+1:

**Pull request merged**